### PR TITLE
just call validate rather than invoking

### PR DESCRIPTION
**Pull request by sandlerr**
### Description

`zat create --no-install` crashes because it passes the `--no-install` argument on to `validate`.

cc @zendesk/vegemite 

### References

https://zendesk-appdevelopers.slack.com/archives/C0L7VFW9G/p1490251653651834

### CODE REVIEW

File: features/step_definitions/app_steps.rb [hunk 1]
```diff
@@ -49,8 +55,15 @@
   FileUtils.ln_s(src, dest)
 end
 
-When /^I run the command "(.*?)" to (validate|package|clean) the app$/ do |cmd, _action|
-  IO.popen(cmd, 'w+') do |pipe|
+When /^I run the command "(.*?)" to (validate|package|clean|create) the app$/ do |cmd, _action|
+  if cmd.start_with? 'zat create'
```
Comment (danielbreves) [inline features/step_definitions/app_steps.rb]: Can extract this to a named method? `stub_api_requests`?

File: features/support/helpers.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+def prepare_env_hash_for(cmd)
+  if cmd.start_with? 'zat create'
+    path = File.expand_path('../../support/webmock', __FILE__)
+    { 'RUBYOPT' => "-r #{path}" }
```
Comment (danielbreves) [inline features/support/helpers.rb]: This seems to be a bit more specific than what the method name implies, no?

File: features/support/webmock.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+unless defined?(Cucumber)
```
Comment (danielbreves) [inline features/support/webmock.rb]: Worth a comment explaining why this is here?

File: features/support/helpers.rb (hunk 2 above)
Comment (sandlerr) [inline features/support/helpers.rb]: ? it returns a hash of environment variables :/

Comment (danielbreves) [inline features/support/helpers.rb]: It does, but only one specific env var for one specific command.

Comment (sandlerr) [inline features/support/helpers.rb]: yes, we lost this information when we chose to extract it.

`prepare_env_hash_for_stubbing_requests_in_zat_create` ?

**Pull request merged**